event_survey/wizard/event_survey_wizard.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, _
+
+
+class WizardEventSurvey(models.TransientModel):
+
+    _name = 'wizard.event.survey'
+
+    survey_template_id = fields.Many2one(
+        comodel_name='survey.survey',
+        reqired=True,
+        domain=[('is_template', '=', True)]
+    )
+    survey_title = fields.Char(
+        string="Survey name",
+    )
+    event_id = fields.Many2one(
+        comodel_name='event.event',
+        default=lambda self: self._default_event(),
+        required=True
+    )
+
+    def _default_event(self):
+        return self.env['event.event'].browse(self.env.context.get(
+            'active_id'))
+
+    def action_generate(self):
+        """Create survey from survey template """
+        self.ensure_one()
+        new_survey = self.survey_template_id.copy()
+
+        # title - html field couldn't be rewrite w/ default, write directly
+        new_survey.write({'title': self.survey_title})
```

Comment:
@simahawk 
title - translated field, if user works under other language then English he would see 'template(copy)', if we need it, must do it manually

Reply:
no problem, they use only FR